client/src/components/ChangePassword.jsx

```diff
@@ -50,11 +51,14 @@ export class ChangePassword extends Component {
                     <Input
                       id="check_new_password"
                       value={ this.state.check_new_password}
+                      size={40}
```

Comment:
Why do you restrict user by 40 symbols?

Reply:
thought that made state size off input

Reply:
This input is limited by col size, so size attribute doesn't impact on my screen. (But on the screen with a high resolution this input can seem broken)